packages/components/src/Form/tests/chromatic/Form.stories.tsx

```diff
@@ -77,15 +79,14 @@ const Template = () => {
             <Checkbox>
                 Agree to terms and conditions
             </Checkbox>
-            <Select isRequired
-                fieldChildren={
-                    <Label>Choose an animal</Label>
-                }
-            >
-                <Select.Option id="designer">Designer</Select.Option>
```

Comment:
[Question] So we were pretty set on not using Select.Option naming? Or we're just naming it as the rest right now before we make a decision?

Reply:
If not, the rest is muchhh cleaner! 👌

Reply:
If not, the rest is muchhh cleaner! 👌

Reply:
ya I don't know the final decision, but I figured if we do change the naming, it would be for all components and we'd change it all at once.